Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.6
+FROM ruby:2.7.1
```

Comment:
Where is this getting pulled from? Strangely I couldn't find a 2.7.1 image on [docker hub](https://hub.docker.com/_/ruby) 🤔 only found 2.6.6 and 2.7.2

Reply:
nevermind me, I just tried pulling it locally and it does pull:

```
➜  decidim-docker git:(sandbox) docker run --rm ruby:2.7.1
Unable to find image 'ruby:2.7.1' locally
2.7.1: Pulling from library/ruby
57df1a1f1ad8: Downloading [========>                                          ]  8.174MB/50.4MB
71e126169501: Download complete
1af28a55c3f3: Download complete
```